packages/grafana-ui/src/components/Forms/Checkbox.tsx

```diff
@@ -125,21 +111,33 @@ export const getCheckboxStyles = stylesFactory((theme: GrafanaTheme2) => {
     `,
     checkmark: css`
       display: inline-block;
-      width: ${checkboxSize};
-      height: ${checkboxSize};
+      width: ${theme.spacing(checkboxSize)};
+      height: ${theme.spacing(checkboxSize)};
       border-radius: ${theme.shape.borderRadius()};
-      margin-right: ${theme.spacing(1)};
       background: ${theme.components.input.background};
       border: 1px solid ${theme.components.input.borderColor};
-      position: absolute;
-      top: 2px;
-      left: 0;
+      pointer-events: none;
```

Comment:
Wonder why is it needed? can't see any difference with or without pointer-events enabled here.

Reply:
The actual white check itself was above the checkbox, and would prevent the hover from happening properly.

I've re-organised the layering of them all and its no longer required.